fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryLoadByKeysPlan.java

```diff
@@ -95,6 +94,7 @@ public RecordQueryLoadByKeysPlan(@Nonnull String parameter) {
                 }, store.getPipelineSize(PipelineOperation.KEY_TO_RECORD))
                 .filter(Objects::nonNull)
                 .map(store::queriedRecord)
+                .map(QueryResult::of)
```

Comment:
I guess should be consistent about whether the `map(of)` happens before or after the `skipThenLimit`, even though it doesn't matter.

Reply:
Right - was trying to get all the map(of) to be last. Done.